src/utils/registerTooltip.jsx

```diff
@@ -0,0 +1,56 @@
+import Chart from 'chart.js';
+
+const handleOnClick = (_, items) => {
+    Chart.lastClickedDataPoint = null;
+    // Check if the click happened on empty chart space or data point
+    if (items.length > 0) {
+        // A Chart element has a reference to which chart it belongs to and what
+        // the current tooltip in view has a reference to the last active tooltip
+        // eslint-disable-next-line no-underscore-dangle
+        Chart.lastClickedDataPoint = items[0]._chart.tooltip._lastActive[0];
+        Chart.helpers.each(Chart.instances, (chartItem) => {
```

Comment:
Could you try moving this block outside of the `if` block?

I believe it would fix the issue that a tooltip will *only* go away if you click on another data point.
It does not work if you click in the white area of the chart or outside of the chart.

Reply:
Yes, it's working! 
Right now when we click on white space tooltip doesn't go away unless we move the mouse.